lib/engine/game/base.rb

```diff
@@ -75,7 +75,7 @@ def self.load(data, at_action: nil, actions: nil, pin: nil, optional_rules: nil,
     class Base
       attr_reader :raw_actions, :actions, :bank, :cert_limit, :cities, :companies, :corporations,
                   :depot, :finished, :graph, :hexes, :id, :loading, :loans, :log, :minors,
-                  :phase, :players, :operating_rounds, :round, :share_pool, :stock_market, :tile_groups,
+                  :phase, :players, :operating_rounds, :round, :round_info, :share_pool, :stock_market, :tile_groups,
```

Comment:
why make this an attr? seems better as just a method that is computed dynamically

def round_info